tour/widget/label.md

```diff
@@ -12,8 +12,8 @@ redirect_from:
 ---
 
 Widgets are the main components of a Fyne application GUI, they can be
-used in an place that a basic `fyne.CanvasObject` can. They manage user
-interactions but also will always match the current theme.
+used in any place that a basic `fyne.CanvasObject` can. They manage user
+interactions and they will always match the current theme.
```

Comment:
I think it would be better with just “and” to avoid repeating the same word once again in the sentence.

Reply:
👌 Done.